### PR TITLE
Update .gitignore and add support for rspec

### PR DESCRIPTION
**Pull request by shelbyspees**
Also changed double quotes to single quotes in `spec/` directory because that's what's recommended in Ruby.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
 /spec/reports/
+/spec/examples.txt
+/test/tmp/
+/test/version_tmp/
 /tmp/
 /bin/
+Gemfile.lock
+
+## Environment normalization:
+/.bundle/
```
Comment (naftulikay) [inline .gitignore]: We want this checked into version control so everyone has the same config.

Comment (smaslennikov) [reply .gitignore]: Yes, both of these are needed

File: Gemfile.lock [hunk 2]
```diff
@@ -1,11 +0,0 @@
-GEM
```
Comment (smaslennikov) [inline Gemfile.lock]: This file should be not be deleted. It locks our versions.

**Comment by smaslennikov** [comment]
You should also squash your commits.

**Pull request merged**